Sources/PrimerSDK/Classes/Core/Checkout Components/PrimerHeadlessUniversalCheckout.swift

```diff
@@ -190,7 +190,10 @@ public class PrimerHeadlessUniversalCheckout {
     }
     
     public static func getAsset(for paymentMethodType: String, assetType: PrimerAsset.ImageType, userInterfaceStyle: PrimerUserInterfaceStyle? = nil) -> UIImage? {
-        let paymentMethodIdentifier = PrimerPaymentMethodType(rawValue: paymentMethodType)?.paymentMethodIdentifier
+        var paymentMethodIdentifier = PrimerPaymentMethodType(rawValue: paymentMethodType)?.paymentMethodIdentifier
+        if paymentMethodIdentifier == nil {
+           paymentMethodIdentifier = paymentMethodType.components(separatedBy: "_").last
```

Comment:
How would `ADYEN_BANCONTACT_CARD` behave here? Or it would never be `nil` at this point?